tests/unlink/14.t

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+# vim: filetype=sh noexpandtab ts=8 sw=8
+# $FreeBSD$
+
+desc="An open file will not be immediately freed by unlink"
+
+dir=`dirname $0`
+. ${dir}/../misc.sh
+
+echo "1..7"
+
+n0=`namegen`
+n2=`namegen`
```

Comment:
Minor question: why not `n1`?

Reply:
Uhh, good question.  I probably copy/pasted from another file.